21688 gh known issue (#8178)
open-cluster-management/rhacm-docs: release_notes/known_issues_global.adoc

Review the known issues for the {global-hub} Operator. The following list contains known issues for this release, or known issues that continued from the previous release. For your {ocp-short} cluster, see link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/release_notes/#ocp-4-15-known-issues[{ocp-short} known issues].

[#hub-cluster-recreates-addon]
== Detached hosted managed hub cluster recreates the _addon_ 

> //2.14:21688

Reviewer: We need a link to the dev bug at the end of the topic and I think we don't need to add these lines below the title anymore.

> When a hosted managed hub cluster gets detached from the {global-hub} cluster, the `detach logic` deletes the `open-cluster-management-agent-addon` namespace and all the `addons` within it. The `local-cluster` addon manager finds these deleted `addons` and recreates them. 

Reviewer: ```suggestion
When a hosted managed hub cluster gets detached from the {global-hub} cluster, the `detach logic` deletes the `open-cluster-management-agent-addon` namespace and all the `addons` within it. The `local-cluster` add-on manager finds the deleted `addons` and recreates them. 
```

Review the known issues for the {global-hub} Operator. The following list contains known issues for this release, or known issues that continued from the previous release. For your {ocp-short} cluster, see link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/release_notes/#ocp-4-15-known-issues[{ocp-short} known issues].

[#hub-cluster-recreates-addon]
== Detached hosted managed hub cluster recreates the _addon_ 
//2.14:21688

> When a hosted managed hub cluster gets detached from the {global-hub} operator, the hub cluster recreates the `addons` in the `open-cluster-management-agent-addon` namespace. 

Author: @ldpliu am I correct that the hub cluster recreates the addons in the open cluster agent namespace? or does something else force it to get recreated?

Reviewer 2: > @ldpliu am I correct that the hub cluster recreates the addons in the open cluster agent namespace? or does something else force it to get recreated?

The detach logic will delete the addon namespace and all addon in this namespace, then the `local-cluster addon manager` found these addon deleted, it will recreate the addons.
This is the bug in `detach logic`,  foundation team will fix it.

> There is currently no workaround for this issue. 

Author: If there are workaround steps, please let me know if there are workaround steps for me to provide :)

Reviewer 2: It should be `detached from the {global-hub} cluster` instead of `detached from the {global-hub} operator`
